filament/backend/src/vulkan/PlatformVkCocoaTouch.mm

```diff
@@ -57,13 +65,12 @@
     createInfo.pNext = NULL;
     createInfo.flags = 0;
     createInfo.pView = metalLayer;
-    VkResult result = vkCreateIOSSurfaceMVK((VkInstance) instance, &createInfo, VKALLOC, &surface);
```

Comment:
You can get rid of the `VKSurfaceKHR surface` declaration above

Reply:
done